src/shim/telemetryClient.ts

```diff
@@ -139,7 +143,6 @@ export class TelemetryClient {
         const startTime = telemetry.time || new Date();
         const endTime = startTime.getTime() + telemetry.duration;
 
-        // TODO: Change resourceManager if ID is provided?
```

Comment:
Don't believe this is necessary. If a request id is passed to `trackRequest` it will be tracked via the customProperties (attributes).

Reply:
As a follow up to this, I don't see any Semantic Resource Attributes in the exporter that would correspond to a request ID.